qf/submission.go

```diff
@@ -14,10 +14,10 @@ func (s *Submission) IsApproved() bool {
 // NewestSubmissionDate returns the submission's build date if newer than the provided submission date.
 // Otherwise, the provided submission date is returned, i.e., if it is newer.
 func (s *Submission) NewestSubmissionDate(submissionDate time.Time) (t time.Time, err error) {
-	if s == nil || s.BuildInfo == nil {
+	if s == nil || s.BuildInfo == nil || s.BuildInfo.BuildDate == nil || s.BuildInfo.SubmissionDate == nil {
```

Comment:
Do we need both submissions date and build date here?

Reply:
Not really, removed the build date one.